libs/s25main/Settings.h

```diff
@@ -104,6 +105,7 @@ class Settings : public Singleton<Settings, SingletonPolicies::WithLongevity>
     {
         unsigned autosave_interval;
         bool revert_mouse;
+        bool enableWindowPinning;
```

Comment:
I want to rename this to just `windowPinning`. Shorter and leaves the door open if `true`/`false` aren't enough. (For example, to add a third value for closing on right-click.)

Reply:
On second thought, doesn't make much of a difference. The current setting names are a mix between imperative mood, past participles, and noun phrases … it's not going to be more consistent one way or the other.

Reply:
Yes I know it's a bit "grown historically"...